src/test/java/com/tuplejump/stargate/cassandra/BasicIndexTest.java

```diff
@@ -39,6 +40,9 @@ public BasicIndexTest() {
         cassandraCQLUnit = CQLUnitD.getCQLUnit(null);
     }
 
+    List<Record> recordsForRow = new ArrayList<Record>();
```

Comment:
Instead of defining these here, can `createTableAndIndexForRowNulls` and `createTableAndIndexForRow` return the `List<Record>`?